app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Person.cs

```diff
@@ -38,9 +38,12 @@ public class Person
         public IEnumerable<Vehicle> Vehicles { get; set; }
         public IEnumerable<OtherAsset> OtherAssets { get; set; }
         public IEnumerable<CompensationClaim> CompensationClaims { get; set; }
-
         public IEnumerable<InsuranceClaim> InsuranceClaims { get; set; }
-
         public string Notes { get; set; }
     }
+
+    public class FoundPerson : Person
```

Comment:
can we use the name PersonFound so it's different from what is coming from the schema to avoid conflicts